lms/views/predicates/_lti_launch.py

```diff
@@ -178,6 +178,13 @@ def __call__(self, context, request):
         return ("canvas_file" in request.params) == self.value
 
 
+class VitalSourceBook(Base):
+    name = "vitalsource_book"
+
+    def __call__(self, context, request):
+        return ("vitalsource_book" in request.params) == self.value
```

Comment:
This check is consistent with how the `canvas_file` predicate above works, but note that I'm not checking for the existence of the required `book_id` and `cfi` parameters here. Similarly the `canvas_file` predicate above doesn't check for the required `file_id` parameter. Where is the best place to put such validation?